subprojects/docs/src/docs/userguide/dep-man/04-modeling-features/variant_model.adoc

```diff
@@ -157,6 +203,136 @@ As they indicate, this is where the difference is made between what needs to be
 It also shows _secondary_ variants, which are exclusive to Gradle projects and not published.
 For example, the secondary variant `classes` from `apiElements` is what allows Gradle to skip the JAR creation when compiling against a <<java_library_plugin.adoc#sec:java_library_classes_usage,`java-library` project>>.
 
+=== The Resolvable Configurations Report
+
+Gradle also offers a complimentary report task called `resolvableConfigurations` that displays the _resolvable_ configurations of a project, which are those which can have dependencies added and be resolved.  The report will list their attributes and any configurations that they extend.  It will also list a summary of any attributes which will be affected by <<variant_attributes.doc#sec:abm_compatibility_rules, Compatibility Rules>> or <<variant_attributes.doc#sec:abm_disambiguation_rules, Disambiguation Rules>> during resolution.
```

Comment:
> which can have dependencies added

Is this correct? Rather, should we be recommending this? I'm not sure if recommending people add dependencies to `compileClasspath` over `implementation` is a good idea, for example.

Reply:
Hmmm.  Good point.  It's technically true that they could be added and resolved, but also we don't want to recommend the legacy ones are actually used this way.  I'll see if I can clarify this.

Reply:
This was marked as resolved but didn't seem to change -- did you not figure out a better wording?